doc/todo/NEWS.md

```diff
@@ -102,33 +113,40 @@ These notes are of interest for people maintaining packages of Elektra:
 
 ## Infrastructure
 
+### Jenkins
+
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
+### Travis
 
-## Website
+- <<TODO>>
+- <<TODO>>
+- <<TODO>>
 
-The website is generated from the repository, so all information about
-plugins, bindings and tools are always up to date. Furthermore, we changed:
+
+## Infrastructure
 
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
+## Compatibility
 
-## Notes for Elektra's Developers
-
-These notes are of interest for people developing Elektra:
+As always, the ABI and API of kdb.h is fully compatible, i.e. programs
+compiled against an older 0.8 version of Elektra will continue to work
+(ABI) and you will be able to recompile programs without errors (API).
 
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
 
 
-## Fixes
+## Website
```

Comment:
Doesn't Website belong to the same category as Documentation?

Reply:
While the website also provides documentation for Elektra, I think website news can include other items, such as updates to the snippet service too.